public/app/features/query-library/api/mappers.ts

```diff
@@ -34,6 +37,11 @@ export const convertAddQueryTemplateCommandToDataQuerySpec = (
         variables: {},
         properties: dataQuery,
       })),
+      user: {
```

Comment:
What I would like to do ideally is purely on the backend... get the user based on who is sending the request

Reply:
Based on this it's still to be implemented in the backend but at least we should not send it from frontend (to avoid tampering with the value)

Reply:
After talking with @kylebrandt , turns out there is no actual bespoke code for adding an entry to the query library - it is all handled via the app platform. There is no place we can currently hook into to add in User information to the backend. We should discuss this during sync to figure out if we make a ticket for it, just ask the channel, or something else to add some middleware to make this possible.